src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -725,11 +727,24 @@ export function registerTerminalActions() {
 		keybinding: {
 			primary: KeyMod.CtrlCmd | KeyCode.End,
 			linux: { primary: KeyMod.Shift | KeyCode.End },
-			when: sharedWhenClause.focusInAny_and_normalBuffer,
+			when: ContextKeyExpr.or(sharedWhenClause.focusInAny_and_normalBuffer, accessibleViewCurrentProviderId.isEqualTo(AccessibleViewProviderId.Terminal)),
 			weight: KeybindingWeight.WorkbenchContrib
 		},
 		precondition: sharedWhenClause.terminalAvailable,
-		run: (xterm) => xterm.scrollToBottom()
+		run: (xterm, accessor) => {
```

Comment:
How about keeping the simple old base functionality and overriding it via context key in the accessibility terminal contrib?

Reply:
I don't think I can register the action with the same ID 🤔

Reply:
If we make the condition a11y buffer is not showing, will it fall back to the editor's keybindings? That would be even better

Reply:
That would not be better IMO because `cmd+up/down arrow` enter/exit the terminal accessible view as opposed to going to the start and end.

Reply:
seems fine to register with the same ID to my surprise.